### DISCUSSION TITLE
Możliwość uzyskania oryginalnego numeru przesyłki z etykiety

### DISCUSSION THREAD

**Discussion by daniel93333** [allegro/allegro-api#7450] [Suggestions - resolved]
Mam taki problem że pakuje paczki za pomocą skanowania kody kreskowego na etykiecie za pomocą którego znajduje konkretne zamówienie.
Problem w tym, że przesyłki allegro one mają inny numer przesyłki niż kod kreskowy na etykiecie - dlatego że przesyłki allegro obsługuje UPS.

Sugeruję możliwość dodania dodatkowego pola, które pozwoli uzyskać informację o oryginalnym numerze przesyłki zawartym w kodzie kreskowym etykiety.

**Comment by MartaNowaczyk** [comment]
Wdrożyliśmy możliwość zwracania takiej informacji [w nowym WZA](https://developer.allegro.pl/documentation/#operation/getShipmentDetails), w obiekcie additionalProperties.

**Comment by daniel93333** [comment]
Dzień dobry,

Nie dowierzan w to co czytam.
Zrobiła mi Pani dzień. Niedowierzałem że w allegro też pracują ludzie... :)

pt., 1 gru 2023, 09:34 użytkownik MartaNowaczyk ***@***.***>
napisał:

**Comment by MILsoldier** [comment]
Dzień dobry,

Czy jest możliwość wyciągnięcia numeru przesyłki UPS (kodu kreskowego z etykiety) dla paczek zwrotnych kurierem Allegro One ? Kod który dostajemy w zwrocie zaczyna się od A...., problem pojawia się przy pikaniu etykiety zwrotnej na przyjęciu paczki na magazynie, przez co wydłuża się proces weryfikacji zwrotu (pozostaje manualne sprawdzenie opisu etykiety i porównanie go z tym co zwróci API). 

Pozdrawiam

**Reply by PrzemyslawLukanowski** [reply]
@MILsoldier Niestety, ale nie udostępniamy takiej opcji. Zachęcam Cię jednak do założenia sugestii takiej zmiany, w nowym wątku, dzięki temu inni użytkownicy również będą mogli zagłosować za takim rozwiązaniem.
